db/install.xml

```diff
@@ -38,6 +38,7 @@
         <INDEX NAME="courseid" UNIQUE="false" FIELDS="courseid"/>
         <INDEX NAME="httpcode" UNIQUE="false" FIELDS="httpcode"/>
         <INDEX NAME="priority_needscrawl" UNIQUE="false" FIELDS="needscrawl, priority"/>
+        <INDEX NAME="url" UNIQUE="false" FIELDS="url"/>
```

Comment:
should this be unique?

Reply:
I'll resolve this convo, not applicable anymore since we can't just add an index here.